include/mbedtls/nist_kw.h

```diff
@@ -0,0 +1,178 @@
+/**
+ * \file nist_kw.h
+ *
+ * \brief This file provides an API for key wrapping (KW)
```

Comment:
There's some funny line wrapping here, and the sentence doesn't end with a full stop.

Reply:
This doesn't on latest version. Please verify you are reviewing latest commits.

Reply:
I take it back, it was fixed by @Patater as part of the commits merge